plugins/generator-1.16.4/forge-1.16.4/procedures/world_data_israining.java.ftl

```diff
@@ -0,0 +1 @@
+((world instanceof World)?((World) world).isRaining():false)
```

Comment:
These would be much better to be implemented with world info like this:

`world.getWorldInfo().isRaining()`

same for thundering (possibly other too?).

Reply:
What's the point? You're adding `getWorldInfo` for nothing...

Reply:
IWorld might not be World in all cases, but worldinfo is available for all classes implementing IWorld ..

That's the purpose of interfaces

Reply:
Okay I see...

Reply:
Done!